templates/default/config.cfg.erb

```diff
@@ -0,0 +1,64 @@
+[general]
+# DNS interface. Note that systemd-resolved may reserve port 53 on 127.0.0.53
```

Comment:
Can we remove all comments and trim this config down a bit? Also add the managed by Chef header.